account_import_line_multicurrency_extension/account_statement_from_invoice_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+    	
+        <record id="view_move_line_tree_bank_statement" model="ir.ui.view">
+            <field name="name">account.move.line.tree.bank.statement</field>
+            <field name="model">account.move.line</field>
+            <field name="arch" type="xml">
+                <tree string="Move lines">
+                    <field name="journal_id" readonly="True"/>
+                    <field name="date" readonly="True"/>
+                    <field name="name" readonly="True"/>
+                    <field name="partner_id" readonly="True" />
+                    <field name="ref" readonly="True" />
+                    <field name="amount_residual_currency" readonly="True"/>
+                    <field name="currency_id" readonly="True"  />
```

Comment:
Generally there is a `groups="base.group_multi_currency"` on the currency fields, should be respected here as well.